sintr_analysis_server_example/bin/decompress.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:path/path.dart';
+import 'package:sintr_worker_lib/instrumentation_transformer.dart';
+
+/// Decompress a compressed instrumentation file.
+main(List<String> args) async {
+  if (args.length != 1) {
+    print('Usage: decompress filePath');
+    exit(1);
+  }
+
+  // Initialization
+  final path = args[0];
+  var srcName = basename(path);
+  if (!srcName.startsWith('compressed-')) throw 'file is already decompressed';
```

Comment:
A lot of our files that are compressed are in a bucket with that name, rather than having it in the file name. If it's decompressed already, it's going to fail in a number of ways below, so I would propose removing this.